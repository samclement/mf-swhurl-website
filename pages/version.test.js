/* eslint-env jest */

import React from 'react'
import { shallow } from 'enzyme'

import Version from './version.js'

describe('With Snapshot Testing', () => {
  it('Version page shows correct "version"', () => {
      const wrapper = shallow(<Version/>)
      expect(wrapper).toMatchSnapshot()
    })
})


