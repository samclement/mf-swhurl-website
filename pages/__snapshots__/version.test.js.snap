// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing Version page shows correct "version" 1`] = `
<p>
  v
  1.0.0
</p>
`;
